src/instructlab/model/backends/backends.py

```diff
@@ -209,7 +209,7 @@ def ensure_server(
     and the URL where it's available."""
 
     try:
-        logger.debug(f"Trying to connect to {api_base}...")
+        logger.info(f"Trying to connect to model server at {api_base}...")
```

Comment:
I think the same comment before about the ... applies here as well?

Reply:
Lol I missed that you're right

Reply:
++done